.gitignore

```diff
@@ -4,3 +4,4 @@ target
 .vscode/settings.json
 .cargo/config.toml
 yarn-error.log
+/.idea
```

Comment:
I don't know this. Looks like some IDE thing. I'm a bit reticent to add such things. IDEs should be able to work without adding their stuff locally. That said, a few lines above I can see `.vscode` which is also an IDE which also seems to not work if it doesn't write stuff locally. So yeah... At least I like that you constrain the exception to the root of the repo.

Reply:
It's JetBrains IDEs, afaik unfortunately they add it all the time whenever you open any project and there is no way to not add it.

Reply:
Yeah, the JetBrains `/.idea` folder is entirely unnecessary [for versions beyond 2019.1](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839)

I could remove this change if necessary, usually `/.idea` folder doesn't get tracked as it can be autogenerated
https://stackoverflow.com/questions/43198273/which-files-in-idea-folder-should-be-tracked-by-git